backend/backend.js

```diff
@@ -41,7 +41,7 @@ app.delete("/pins/:id", async (req, res) => {
 app.put("/pins/upvote/:id", async (req, res) => {
   const id = req.params["id"];
   let pin = await pinServices.findPinById(id);
-  if (pin === undefined) res.status(404).send("Pin not found.");
+  if (pin === undefined) res.status(404).send("Pin type not found.");
```

Comment:
Revert this please.

Reply:
When we upvote a function, we're searching for a specific pin, not a type.

Reply:
Oops, not sure why this changed.